lib/ramble/ramble/cmd/workspace.py

```diff
@@ -1446,6 +1446,134 @@ def workspace_manage_includes(args):
             ws.add_include(args.add_include)
 
 
+def workspace_manage_modifiers_setup_parser(subparser):
+    """manage workspace modifiers"""
+    actions = subparser.add_mutually_exclusive_group()
+    actions.add_argument(
+        "--list", "-l", action="store_true", help="whether to print existing modifiers"
+    )
+
+    actions.add_argument(
+        "--add",
+        action="store_true",
+        help="whether to remove an existing modifier by index",
+    )
+
+    actions.add_argument(
+        "--remove",
+        action="store_true",
+        help="whether to remove an existing modifier by index",
+    )
+
+    subparser.add_argument(
+        "--mod-index",
+        "-i",
+        dest="remove_index",
+        metavar="INDEX",
+        help="index of modifier to remove, only used with --remove",
+    )
+
+    subparser.add_argument(
+        "--scope",
+        "-s",
+        dest="scope",
+        metavar="SCOPE",
+        default=None,
+        help="scope of modifier to manage. Can be a glob when removing. Of the form "
+        "<application_name>:<workload_name>:<experiment_name> or 'workspace' to "
+        "manage workspace modifiers.",
+    )
+
+    subparser.add_argument(
+        "--name",
+        "-n",
+        dest="name",
+        metavar="NAME",
+        default=None,
+        help="name of modifier to manage. Can be a glob when removing, and defaults to '*'. "
+        "Should be the name of a modifier object.",
+    )
+
+    subparser.add_argument(
+        "--mode",
+        "-m",
+        dest="mode",
+        metavar="MODE",
+        default=None,
+        help="mode of modifier to manage. Can be a glob when removing, and defaults to None, "
+        "using the default for the modifier. ",
+    )
+
+    subparser.add_argument(
+        "--on-executable",
+        "-e",
+        dest="on_executable",
+        metavar="EXEC_LIST",
```

Comment:
Do you think something like `nargs='*'` or `nargs='+'` would be more appropriate for the list stuff here?

Reply:
I think this syntax is more consistent with the rest of our manage commands. I also think `'*'` has some issue in how it parses relative to other arguments. `'+'` might be good, but if we're going to make this change we should do it in the other manage commands as well.